fvp-psa-sp.mk

```diff
@@ -128,4 +135,29 @@ carveout-dtb-clean:
 
 boot-img-clean: carveout-dtb-clean
 endif
+
+ifeq ($(TS_FW_UPDATE),y)
+
+# TODO: the fwu-tool is currently not needed.
+$(eval $(call  build-ts-host-app,fwu-tool))
+
+ffa-fwu-sp: ts-host-fwu-tool
+
+# Copy the disk image used by FWU to the build directory to allow the FVP binary to find it.
+$(ROOT)/build/secure-flash.img:
+	cp $(ROOT)/trusted-services/components/media/disk/disk_images/multi_location_fw.img $(ROOT)/build/secure-flash.img
+
+# Add a shortcut to help manually doing the copy.
+ffa-fwu-fash-img: $(ROOT)/build/secure-flash.img
+
+# TODO: find the correct place for this dependency.
+ffa-fwu-sp: $(ROOT)/build/secure-flash.img
```

Comment:
`$(ROOT)/out/bin/secure-flash.img` perhaps?

Reply:
Updated to use the out directory as you proposed.

Reply:
`$(ROOT)/build` is not a good directory, it's better to put it in `$(ROOT)` then. By the way, FVP is executed from the `$(ROOT)/build`, I think it would make sense to cd to `$(ROOT)/out/bin` or `$(ROOT)/out just as we do with QEMU.

Reply:
Thank you for the advice. Corrected.